src/libsync/caseclashconflictsolver.h

```diff
@@ -0,0 +1,95 @@
+#ifndef CASECLASHCONFLICTSOLVER_H
+#define CASECLASHCONFLICTSOLVER_H
```

Comment:
why not use `#pragma once` as elsewhere?

Reply:
this is now done with `#pragma once`